### DISCUSSION TITLE
Older JSCAD files not working

### DISCUSSION THREAD

**Discussion by bubba925** [jscad/OpenJSCAD.org#1372] [Q&A]
Good evening all!

I found some older jscad files from 2020 that I was hoping to get working to convert for 3d printing. It seems the changes since 2020 have broken this code and I am curious if there is an easy way to get it working? I tried some basic troubleshooting but kept encountering new errors as I fixed some. This is the repo I was hoping to get converted to stls. I would appreciate any help but also understand if its too labor intensive to translate the older code.

https://github.com/c-leitner/3d-models/tree/master/server

I did fine this thread but was not sure if these were the only changes required since I ran into more issues in my quick attempts to troubleshoot. https://github.com/jscad/OpenJSCAD.org/discussions/883

**Comment by bubba925** [comment]
Found old jscad instance that worked. openjscad.surge.sh

**Comment by z3dev** [comment]
There's a bridge layer that supports all those crazy inconsistent APIs of OpenSCAD. You can use that package, adopt some of the code, or use the code for inspiration. Hopefully, your designs can be converted easily.

https://github.com/jscad-community/jscad-scad-api
